src/pages/Environments.js

```diff
@@ -539,21 +539,21 @@ export const Environments = ({ nav = undefined }) => {
     }, [makeMenuIcon('trash'), 'Delete'])
   }
 
-  const renderPauseButton = (computeType, compute) => {
+  const renderPauseButton = (computeType, compute, isGcp) => {
     const { status } = compute
 
     const shouldShowPauseButton =
-      Utils.cond(
-        [isApp(compute) && !_.find(tool => tool.appType && tool.appType === compute.appType)(appTools)?.isPauseUnsupported, () => true],
-        [isPauseSupported(getToolLabelFromRuntime(compute)) && currentUser === getCreatorForRuntime(compute), () => true],
-        () => false)
+    Utils.cond(
```

Comment:
Nit: My instinct says this line should either be indented one step or have its initial linebreak removed (so line 545 becomes `const shouldShowPauseButton = Utils.cond(`) - having it at the same level is a bit misleading